corehq/apps/sms/tasks.py

```diff
@@ -462,11 +466,6 @@ def store_billable(self, msg):
             )
         except RetryBillableTaskException as e:
             self.retry(exc=e)
-        except DeliveredBillableException:
```

Comment:
excellent! we might want to add a comment in this task near the `self.retry(exc=e)` that is something along the lines of
```
WARNING: do not manually remove messages from the queue unless you have a backup plan for how to processes them before the end of the month billing cycle. If not, LEAVE AS IS.
```
Thank you so much for catching this.

Reply:
Good idea